mode/examples/Topics/Animation/Sequential/Sequential.pyde

```diff
@@ -1,47 +1,45 @@
 """
 Sequential
-by James Paterson.    
+by James Paterson.
 
-Displaying a sequence of images creates the illusion of motion. 
-Twelve images are loaded and each is displayed individually in a loop. 
+Displaying a sequence of images creates the illusion of motion.
+Twelve images are loaded and each is displayed individually in a loop.
 """
 
 numFrames = 12  # The number of frames in the animation
-currentFrame = 0
 images = [PImage] * numFrames
 
 def setup():
     size(640, 360)
     frameRate(24)
 
-    images[0] = loadImage("PT_anim0000.gif")
-    images[1] = loadImage("PT_anim0001.gif")
-    images[2] = loadImage("PT_anim0002.gif")
-    images[3] = loadImage("PT_anim0003.gif")
-    images[4] = loadImage("PT_anim0004.gif")
-    images[5] = loadImage("PT_anim0005.gif")
-    images[6] = loadImage("PT_anim0006.gif")
-    images[7] = loadImage("PT_anim0007.gif")
-    images[8] = loadImage("PT_anim0008.gif")
-    images[9] = loadImage("PT_anim0009.gif")
-    images[10] = loadImage("PT_anim0010.gif")
-    images[11] = loadImage("PT_anim0011.gif")
+    images[0] = loadImage("data/PT_anim0000.gif")
```

Comment:
#113
The issue name maybe isn't the greatest, but I was trying to dig deeper than "standalone doesn't respect the `data` dir".